tsfresh/feature_extraction/feature_calculators.py

```diff
@@ -631,6 +631,23 @@ def standard_deviation(x):
     return np.std(x)
 
 
+@set_property("fctype", "simple")
+def variation_coefficient(x):
+    """
+    Returns the variation coefficient of x
+
+    :param x: the time series to calculate the feature of
+    :type x: numpy.ndarray
+    :return: the value of this feature
+    :return type: float
+    """
+    mean = np.mean(x)
+    if mean != 0:
+        return np.std(x) / np.mean(x)
```

Comment:
You could save some computing time (even though not really needed) by using your already calculated `mean` here